src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/EnvelopeRepositoryTest.java

```diff
@@ -122,13 +122,16 @@ public void should_get_1_incomplete_envelope_from_yesterday() {
             envelope("D.zip", "Z", Status.COMPLETED)
         );
 
-        // and update createAt to yesterday
+        // and update createAt to 2h ago
         entityManager.createNativeQuery(
-            "UPDATE envelopes SET createdat = '" + now().minusDays(1) + "' WHERE zipfilename IN ('A.zip', 'B.zip')"
+            "UPDATE envelopes SET createdat = '" + now().minusHours(2) + "' WHERE zipfilename IN ('A.zip', 'B.zip')"
```

Comment:
I know it's already merged, but here's a suggestion: how about making it 61 minutes ago instead of 2 hours? This would eliminate a potential time zone discrepancy issue.